modules/betweenBidAdapter.js

```diff
@@ -59,9 +59,9 @@ export const spec = {
       if (i.params.itu !== undefined) {
         params.itu = i.params.itu;
       }
-      if (i.params.cur !== undefined) {
-        params.cur = i.params.cur;
-      }
+
+      params.cur = i.params.cur || 'USD';
+
```

Comment:
Did you want to remove the not undefined check for the currency parameter? Other params in this adapter are  still checking inside if blocks.

Reply:
Hi, Chris. Thanks for your comment, yes you're right, I need the "cur" parameter to have a default value of "USD" if it was not defined and it should always be contained in the bid request.

Reply:
@ChrisHuie, is everything ok? Do I need to fix something in the code?